fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -1063,12 +1062,15 @@ protected DataCache<?> buildCache() {
      *
      * @return the instance of local signature cache
      */
-    protected DataCache<?> buildLocalSignatureCache() {
+    protected TupleDataCache<String, ?, ?> buildLocalSignatureCache() {
```

Comment:
Shouldn't this return a `TupleDataCache<String, Long, String>` rather than a `<String, ?, ?>`? It seems like the specific type is known in all the paths.